orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -151,9 +152,18 @@ class TrainingRunner {
 
     int data_parallel_size = 1;
     int horizontal_parallel_size = 1;
-    int pipeline_stage_size = 1;
     // Enable gradient clipping.
     bool enable_grad_norm_clip=true;
+
+    // Pipeline configuration.
+
+    // model_paths[i] is the name of the i-th pipeline stage's forward model.
+    // The i-th stage is run by the i-th MPI rank. 
```

Comment:
This will not be true once data parallelism is enabled for pipeline. Maybe add a comment for that?

Reply:
I change the way we load partition files. Here is the new comment:
```
    // model_paths[i] is the name of the pipeline stage for i-th process.
    // The i-th file is run by the i-th MPI rank. 
    // If model_paths is not empty, model partition transformation may not be internally invoked.
```
This is more general.